### PR TITLE
final project

### PR DESCRIPTION
**Pull request by paxtonfitzpatrick**

### CODE REVIEW

File: data-stories/semantic-mapping/utils.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        os.listdir(dir)
+        lstime = time.time() - stime
+        if lstime < maxtime:
+            print ("Disk access is quick (%0.3f seconds to ls), continuing.." % lstime)
+            return
+        else:
+            print ("Disk access is slow (%0.3f seconds to ls), waiting more.." % lstime)
+            time.sleep(retrytime)
+
+    print ("Disk access is slow but fuck it, I'm starting anyway..")
```
Comment (jeremymanning) [inline data-stories/semantic-mapping/utils.py]: could you edit this statement to be more "family friendly" ?

**Comment by jeremymanning** [comment]
hey @paxtonfitzpatrick one minor edit request (editing a print statement).  also, do you need the __pycache__ files to be part of the commit?

**Comment by paxtonfitzpatrick** [comment]
@jeremymanning 
whoops, I meant to add the pycache files to .gitignore.  Want me to do that for the full repo or just this project's directory?

I don't see the edit request anywhere -- which print statement do you want me to edit?

**Comment by jeremymanning** [comment]
Re: edit, line 222 of data-stories/semantic-mapping/utils.py

Could you add all pycache files to gitignore and also remove them from this pull request?

**Comment by paxtonfitzpatrick** [comment]
Yikes, sorry! The file with that print statement actually part of the module Huth et al. wrote to handle the analysis pipeline. I'll make a quick edit and also deal with the pycache files.

**Comment by paxtonfitzpatrick** [comment]
@jeremymanning should be good to go now

**Pull request merged**